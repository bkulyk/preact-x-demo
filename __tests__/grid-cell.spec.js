import React from 'react';
import GridCell from '@bitchin/grid-cell';
import { shallow } from './enzyme';
import { it, expect } from './jest'

it('should render correctly', () => {
  expect(shallow((
    <GridCell
      span={4}
      className="whatever"
      order={2}
      data-text="test"
    />
  ))).toMatchSnapshot();
});

it('should render with with phone and fixed width', () => {
  expect(shallow((
    <GridCell
      phone={12}
      tablet={4}
      desktop={2}
      align="middle"
      className="custom"
      fixedWidth
    />
  ))).toMatchSnapshot();
});