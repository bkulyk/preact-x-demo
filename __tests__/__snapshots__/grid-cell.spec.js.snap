// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<div
  className="whatever mdc-layout-grid__cell mdc-layout-grid__cell--span-4 mdc-layout-grid__cell--order-2 mdc-layout-grid__cell--align-top"
  data-text="test"
/>
`;

exports[`should render with with phone and fixed width 1`] = `
<div
  className="custom mdc-layout-grid__cell mdc-layout-grid__cell--align-middle mdc-layout-grid__cell--span-2-desktop mdc-layout-grid__cell--span-4-tablet mdc-layout-grid__cell--span-12-phone mdc-layout-grid--fixed-column-width"
/>
`;
